### DISCUSSION TITLE
PCBox - use Pentium III Tualatin 1400MHz will cause Win2000 BSOD

### DISCUSSION THREAD

**Discussion by Fubuki0x10DE** [PCBox/PCBox#83] [General]
Use Socket 370 Pentium III Tualatin processor will cause Win2000 BSOD.
0x0000001E KMODE_EXCEPTION_NOT_HANDLED

**Comment by bachimusprime** [comment]
Why did you make this a discussion and not an issue on the issue tracker? Also, I've never had this issue with the Katmai or Coppermine Pentium 3 CPUs emulated, so not sure why Tualatin would be an issue lol. Also, Tualatin doesn't even run full speed on any CPU out there anyway, so not sure why you're using it lol
